docs/data_quality.md

```diff
@@ -58,6 +58,7 @@ The definition of "impact" will vary depending on how fine-grained the informati
 
 * Where relevant, an `alias` to the equivalent CVE record is present
 * Where an OSV record consolidates multiple vulnerabilities in another ecosystem (or universe), multiple `related` identifiers are present
+* Where an OSV record derives from another record, relevant `upstream` identifiers are present.
```

Comment:
I'd be more specific here and say something along the lines of...

Where an OSV record describes a vulnerability which is downstream of a vulnerability in another record (e.g. a Linux distribution repackaging an upstream tool)... 

(probably needs a bit of tweaking/wordsmithing to make this clearer)